docs/index.md

```diff
@@ -165,6 +166,11 @@ $ eval "$(docker-machine env dev)"
 $ docker ps
 ```
 
+> **Note**: If you are using `fish`, or a Windows shell such as
+> Powershell/`cmd.exe` the above method will not work as described.  Instead,
+> see [the `env` command's documentation](http://docs.docker.com/machine/#env)
```

Comment:
Or just `/machine/#env`, if it's no problem that the link doesn't work when reading on GitHub